src/module-elasticsuite-indices/Model/Healthcheck/ReplicasConfigCheck.php

```diff
@@ -66,7 +67,7 @@ public function __construct(
         UrlInterface $urlBuilder,
         int $sortOrder = 30
     ) {
-        parent::__construct($urlBuilder, $sortOrder);
+        parent::__construct($urlBuilder, $sortOrder, MessageInterface::SEVERITY_CRITICAL);
```

Comment:
Please add the severity is as an exposed constructor parameter, after the sortOrder.